extra_geom/pyfai.py

```diff
@@ -42,3 +42,13 @@ class LPD1M(Detector):
 
     def __init__(self, pixel1=5e-4, pixel2=5e-4, **kwargs):
         super().__init__(pixel1, pixel2, **kwargs)
+
+
+class JUNGFRAU(Detector):
+    IS_CONTIGUOUS = False
+    MAX_SHAPE = (0, 256)  # num modules filled at instantiation
```

Comment:
If it works without this line, I'd get rid of it, so we don't have some deliberatly incorrect information.

Reply:
Sure, removed in https://github.com/European-XFEL/EXtra-geom/pull/197/commits/52b84b06891d7961558c542c16ea12b9a95b5418.